<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWindow</class>
 <widget class="QWidget" name="VideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;
font-size:14px;
color: #ffffff;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_setHome">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">

 QPushButton {
      
		border-image: url(:/images/setHome.png);

  }

  QPushButton:pressed {
		border-image: url(:/images/setHome_p.png);

  }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video_swith">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>370</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton {
background-color: transparent;
		border-image: url(:/images/videoType_swith.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/videoType_swith_p.png);

  }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_replay">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton {
      
		border-image: url(:/images/video_replay.png);

  }

  QPushButton:pressed {
		border-image: url(:/images/video_replay_p.png);

  }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_cloudControl" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:/images/cloudControl_bg1.png);</string>
    </property>
    <widget class="QPushButton" name="pushButton_cloudControl_up">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
 QPushButton {
      
		border-image: url(:/images/cloudControl_up.png);

  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_up_p.png);

  }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cloudControl_down">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>138</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
 QPushButton {
		border-image: url(:/images/cloudControl_down.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_down_p.png);
  }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cloudControl_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
 QPushButton {
		border-image: url(:/images/cloudControl_left.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_left_P.png);
  }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cloudControl_right">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
 QPushButton {
		
		border-image: url(:/images/cloudControl_right.png);
  }

  QPushButton:pressed {
		border-image: url(:/images/cloudControl_right_p.png);
  }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_videoLoss">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>283</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/videoloss.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_recordState">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>57</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/record_state.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_motionDetection">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/motetion_warn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ipc_identify">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>92</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
